statchatui/src/Home/Home.jsx

```diff
@@ -46,19 +41,7 @@ const Home = ({
       platform,
       gameName,
     };
-    try {
-      await fetch(`http://localhost:3002/users/profile`, {
-        method: "POST",
-        headers: {
-          "Content-Type": "application/json",
-        },
-        body: JSON.stringify(formData),
-        credentials: "include",
-        //this includes any cookies or credentials the server might need to validate the user, without this, the user will not be identified in the post request and will not update the correct user in the database
-      });
-    } catch (error) {
-      console.error("Error updating profile:", error);
-    }
+    postToDatabase(formData);
```

Comment:
This should have a more specific name as well

Reply:
i added a comment for it